lib/content-services/src/lib/category/categories-management/categories-management.component.scss

```diff
@@ -11,6 +17,24 @@
         .adf-btn-padded {
             margin-right: -14px;
         }
+
+        .mat-form-field-underline {
+            display: none;
+        }
+
+        .mat-form-field-wrapper {
```

Comment:
this is incorrect, you are hacking the internal styles of the component, should be like this:

```css
.mat-form-field {
            width: 100%;
        }
```